logging/tests/system/test_system.py

```diff
@@ -179,7 +179,8 @@ def test_log_text_with_resource(self):
         logger = Config.CLIENT.logger(self._logger_name("log_text_res"))
         now = datetime.datetime.utcnow()
         resource = Resource(
-            type="gae_app", labels={"module_id": "default", "version_id": "test"}
+            type="gae_app",
+            labels={"module_id": "default", "version_id": "test", "zone": ""},
```

Comment:
This is something that was already failing on `master`, does not seem to be caused by version bumps in this PR.
(I did not research when exactly the failure was introduced, though)

Reply:
I remember seeing something about this. It sounds like it changed fairly recently in the backend.